README.md

```diff
@@ -0,0 +1,78 @@
+# SSV Distributed Key Generation Spec
+
+## Introduction
+This repository contains spec of distributed key generation (DKG) used in SSV. Based on verifiable secret sharing, DKG allows a set of *n* operators to generate a  **BLS** key share for each operator, such that a BLS private key can only be used when threshold *t* out of *n* operators agree. 
+
+---
+
+## Functionalities
+Operations are initiated by an initiator, and executed by operators. An initiator can initiate:
+
+### Init
+When a init message is sent to the operators, all operators participate and run a DKG ceremony to create key shares. A DKG ceremony asks each operator to verifiably share locally generated secrets with other operators, and combine secret shares from all operators to derive its own BLS key share. After the ceremony, each operator generates a **Result** with a **SignedProof** (see below) for verification. Proofs are referred to when an initiator asks for re-sign or reshare. 
+
+### Re-sign
+In the case where the nonces of a DKG ceremony is not correct, to prevent replay attack, the initiator sends re-sign messages for operators to create new signatures with correct nonces without generating new key shares. 
```

Comment:
It is creating new key-shares, though. The procedure only changes the `signature` part of the shares...but it does change the overall `sharesData`.
Re-phrase to "without generating a new validator key"? 🤔

Reply:
Also, the signature part is `owner-nonce`, so it could be that both were wrong...for whatever reason. Might be just more "complete" to clarify that both can be changed.